area4/__init__.py

```diff
@@ -65,7 +65,16 @@ def divider(self, number) -> str:
                 return self.dividers[number].split("\n")[0]
             except IndexError:
                 raise ValueError('That divider doesn\'t exist!')
-
+    def logDivider(self, div, *stuff) -> None:
+        if(type(div) is int) div = self.dividers[div]
+        if len(stuff) == 1:
+            print(stuff[0])
+            return
+        for s in stuff:
+           print(s)
```

Comment:
Please avoid using print, return is preferred. If you must, make sure you add a return type annotation of None.

Reply:
I did!